lib/maropost/contact.rb

```diff
@@ -0,0 +1,91 @@
+require 'active_model'
```

Comment:
same comment if you move this to moropost.rb you can avoid this on other files too